CHANGELOG.rst

```diff
@@ -1,5 +1,42 @@
 .. _changelog:
 
+
+0.22.0
+------
+
+Renku ``0.22.0`` improves the Renku experience when working on RenkuLab.
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+* 🔲 **UI**: Improve styling of buttons with menu and group buttons
+  (`#2243 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2243>`_).
+* 👤 **UI**: Limit the available namespaces for new projects to owned-only to prevent failures
+  (`#2187 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2187>`_).
```

Comment:
This sounds like a bug to me? Also, are you referring to namespaces available at project creation? If so, I would reword to `Limit the available namespaces in which new projects may be created to one ones owned by the user to prevent failures`